src/crate/client/http.py

```diff
@@ -82,13 +83,17 @@ def super_len(o):
 
 class CrateJsonEncoder(json.JSONEncoder):
 
-    epoch = datetime(1970, 1, 1)
+    epoch_aware = datetime(1970, 1, 1, tzinfo=pytz.UTC)
```

Comment:
```suggestion
    epoch_aware = datetime(1970, 1, 1, tzinfo=timezone.utc)
```

I think it's available in all python versions we support: https://docs.python.org/3/library/datetime.html?highlight=timezone#datetime.timezone.utc

Could drop the pytz dependency.

Reply:
Ah sure, I missed that. Thank you. Adjusted the patch.